diff --git a/scripts/reanimated_utils.rb b/scripts/reanimated_utils.rb
index ec240a0c099383c6b421a341364b6b4d628cab85..8260066ea3b32ea2ba8dc63345047fff2c580d42 100644
--- a/scripts/reanimated_utils.rb
+++ b/scripts/reanimated_utils.rb
@@ -17,7 +17,7 @@ def find_config()
     :react_native_reanimated_dir_from_pods_root => nil,
   }
 
-  react_native_node_modules_dir = File.join(File.dirname(`cd "#{Pod::Config.instance.installation_root.to_s}" && node --print "require.resolve('react-native/package.json')"`), '..')
+  react_native_node_modules_dir = File.join(File.dirname(`cd "#{Pod::Config.instance.installation_root.to_s}/rn-app" && node --print "require.resolve('react-native/package.json')"`), '..')
   react_native_json = try_to_parse_react_native_package_json(react_native_node_modules_dir)
 
   if react_native_json == nil
