diff --git a/RNReanimated.podspec b/RNReanimated.podspec
index dbc740333617894ed0649d10214b4f0a28af0910..0992170482bc56dfbe7b31268aad5a937392ed53 100644
--- a/RNReanimated.podspec
+++ b/RNReanimated.podspec
@@ -133,6 +133,8 @@ Pod::Spec.new do |s|
   s.dependency "ReactCommon/turbomodule/core"
   if defined?(install_modules_dependencies()) != nil
     install_modules_dependencies(s)
+    s.dependency 'React-jsi'
+    s.dependency 'React-hermes'
   else
     install_modules_dependencies_legacy(s)
   end
diff --git a/scripts/reanimated_utils.rb b/scripts/reanimated_utils.rb
index 9fc7b15b617754cf698f9a973ea4666052bc8199..e4d74ce65836c2ee53bd130b3f48057a59831944 100644
--- a/scripts/reanimated_utils.rb
+++ b/scripts/reanimated_utils.rb
@@ -17,7 +17,7 @@ def find_config()
     :react_native_reanimated_dir_from_pods_root => nil,
   }
 
-  react_native_node_modules_dir = File.join(File.dirname(`cd "#{Pod::Config.instance.installation_root.to_s}" && node --print "require.resolve('react-native/package.json')"`), '..')
+  react_native_node_modules_dir = File.join(File.dirname(`cd "#{Pod::Config.instance.installation_root.to_s}/rn-app" && node --print "require.resolve('react-native/package.json')"`), '..')
   react_native_json = try_to_parse_react_native_package_json(react_native_node_modules_dir)
 
   if react_native_json == nil
